assets/js/autocomplete/suggestions.js

```diff
@@ -131,14 +144,30 @@ function childNodeSuggestion (childNode, parentId, query, category, label) {
  * Builds a suggestion for a node section.
  */
 function nodeSectionSuggestion (node, section, query, category, label) {
-  if (!matchesAny(section.id, query)) { return null }
+  if (!matchesAny(section.title || section.id, query)) { return null }
```

Comment:
I am not the biggest fan of the added conditionals. It makes the code hard to follow. Can't we normalize them to be as close as possible when generating the search data from Elixir side?